src/FixerDefinition/CodeSample.php

```diff
@@ -28,13 +28,20 @@
     private $configuration;
 
     /**
+     * @var null|int
+     */
+    private $requiredVersion;
+
+    /**
      * @param string     $code
      * @param null|array $configuration
+     * @param null|int   $requiredVersion
      */
-    public function __construct($code, array $configuration = null)
+    public function __construct($code, array $configuration = null, $requiredVersion = null)
     {
         $this->code = $code;
         $this->configuration = $configuration;
+        $this->requiredVersion = $requiredVersion;
```

Comment:
Lower bound default can be set to the PHP min. version required by the fixer tool.